### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.4.1 - add headings to tab panels

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1294]
# Feature Request

## Description
Screen reader users navigate pages by way of certain semantic elements (landmarks), nav header and main among them. Each block of content should have its own heading at the appropriate level compared to the surrounding content. 

## Solution
Add a heading h2 to each [tab panel](https://bitv-v12.live.typo3.cps.321.works/content-examples/interactive/tab).

**Comment by alexakressmann** [comment]
@SarahEdding Please have a look today, thank you

**Comment by SarahEdding** [comment]
I've added a[ heading h2 to each tab panel](https://bitv-v12.live.typo3.cps.321.works/content-examples/interactive/tab). This was a content issue.  
The element doesn't technically need one, but headings help understand the content, especially when the visual connect beween tab list and tab panel is lost. 
See screenshot of page without styles: 
![image](https://github.com/benjaminkott/bootstrap_package/assets/132338236/180595f8-6231-4019-b8bf-8e61c4c3afcb)
